### ISSUE TITLE
Mobile - Accessibility Mode - Unable to focus date on single Tap

### ISSUE THREAD

**Issue by vipinuk84** [eureka2/ab-datepicker#4]
Hi,

In mobile accessibility mode, plugin is not supporting selecting the date on single tap. When I accessed the url http://eureka2.github.io/ab-datepicker/, i was able to select the date on single tap.

But in the code downloaded, single tap shift the focus back to the default date. A long press required to keep the focus on any other date. 

Is there anyway to fix this?

Thanks for your help
vipin

**Comment by vipinuk84** [comment]
Interestingly, it worked when I took the DatePicker.js which is used in the website example.
